### PR TITLE
Support profiles

### PR DESCRIPTION
**Pull request by lnielsen**
Update Invenio-UserProfiles to user the new profile fields in Invenio-Accounts

### Checklist

Ticks in all boxes and 🟢 on all GitHub actions status checks are required to merge:

- [x] I'm aware of the [code of conduct](https://inveniordm.docs.cern.ch/contribute/code-of-conduct/).
- [x] I've created [logical separate commits](https://inveniordm.docs.cern.ch/develop/best-practices/commits/#commits) and followed the [commit message format](https://inveniordm.docs.cern.ch/develop/best-practices/commits/#commit-message).
- [x] I've added relevant test cases.
- [x] I've added relevant documentation.
- [x] I've marked [translation strings](https://inveniordm.docs.cern.ch/develop/best-practices/i18n/) (for relevant code).
- [x] I've followed the [CSS/JS](https://inveniordm.docs.cern.ch/develop/best-practices/css-js/) and [React](https://inveniordm.docs.cern.ch/develop/best-practices/react/) guidelines (for relevant code).
- [x] I've followed the [web accessibility](https://inveniordm.docs.cern.ch/develop/best-practices/accessibility/) guidelines (for relevant code).
- [x] I've followed the [user interface](https://inveniordm.docs.cern.ch/develop/best-practices/ui/) guidelines (for relevant code).
- [x] I've identified the [copyright holder(s)](https://inveniordm.docs.cern.ch/contribute/copyright-policy/) and updated copyright headers for touched files (>15 lines contributions).
- [x] I've NOT included third-party code (copy/pasted source code or new dependencies).

**Third-party code**

If you've added [third-party code (copy/pasted or new dependencies)](https://inveniordm.docs.cern.ch/develop/best-practices/commits/#third-party-codedependencies), please reach out to an [architect](https://github.com/orgs/inveniosoftware/teams/architects/members).

**Reminder**

By using GitHub, you have already agreed to the [GitHub’s Terms of Service](https://help.github.com/articles/github-terms-of-service/#6-contributions-under-repository-license) including that:

1. You license your contribution under the same terms as the current repository’s license.
2. You agree that you have the right to license your contribution under the current repository’s license.

### CODE REVIEW

File: invenio_userprofiles/alembic/41157f1933d6_remove_table.py [hunk 1]
```diff
@@ -0,0 +1,47 @@
+#
```
Comment (lnielsen) [inline invenio_userprofiles/alembic/41157f1933d6_remove_table.py:1]: Still need to add a data migration from the existing table to the new table.

File: invenio_userprofiles/views.py [hunk 2]
```diff
@@ -143,19 +145,13 @@ def handle_profile_form(form):
...
-                current_user.email = form.email.data
-                current_user.confirmed_at = None
-                db.session.add(current_user)
+                    form.email.data != current_user.email:
                 email_changed = True
-        db.session.commit()
+            form.populate_obj(current_user)
+
+            db.session.add(current_user)
+        current_app.extensions['security'].datastore.commit()
```
Comment (lnielsen) [inline invenio_userprofiles/views.py:154]: @ppanero FYI: This line is important for your work.

Comment (ppanero) [reply invenio_userprofiles/views.py:154]: This should trigger some of the hooks, that's what you mean?

Comment (lnielsen) [reply invenio_userprofiles/views.py:154]: Yes, it's the same thing. Previously here was only a database session commit, which would have then not caught an update to the users table.

Comment (ppanero) [reply invenio_userprofiles/views.py:154]: Perfect thanks!

**Pull request merged**